test/cpp/lite_interpreter_runtime/test_lite_interpreter_runtime.cpp

```diff
@@ -49,6 +49,18 @@ TEST(RunTimeTest, LoadAndForward) {
   ASSERT_EQ(result, expected_result);
 }
 
+TEST(RunTimeTest, Delegate) {
+  std::string filePath(__FILE__);
+  auto testModelFile = filePath.substr(0, filePath.find_last_of("/\\") + 1);
+  testModelFile.append("delegate_test.ptl");
```

Comment:
> It might also be a good idea to include a script to generate it in case it needs to be updated in the future. You could also commit the generation script that and run that first during the test instead of committing the `.ptl`.

Generating the .ptl is covered in test_backend.cpp. There we have the full jit to compile, generate and run the model. In this test environment we want to run it with minimum binary, and to test the existing model with no backward compatibility break.

Reply:
Maybe just add a comment indicating how to generate it from test_backend.cpp